tests/tensor_search/integ_tests/test_search_combined.py

```diff
@@ -644,3 +642,82 @@ def test_search_vectoriseIsCalledWithEnableCacheTrue(self):
                     args, kwargs = mock_vectorise.call_args
                     self.assertTrue(kwargs["enable_cache"])
                 mock_vectorise.reset_mock()
+
+    def test_empty_lexical_query(self):
+        for index in [self.structured_default_text_index, self.unstructured_default_text_index]:
+            with self.subTest(index=index.type):
+                tensor_search.add_documents(
+                    config=self.config,
+                    add_docs_params=AddDocsParams(
+                        index_name=index.name,
+                        docs=[
+                            {"_id": "1", "text": "document_1"},
+                            {"_id": "2", "text": "document_2"},
+                            {"_id": "3", "text": "document_3"},
+                            {"_id": "4", "text": "document_4"},
+                            {"_id": "5", "text": "document_5"},
+                            {"_id": "6", "text": "document_6"},
+                            {"_id": "7", "text": "document_7"},
+                            {"_id": "8", "text": "document_8"},
+                            {"_id": "9", "text": "document_9", "my_list": ["tag1", "tag2 some"]},
+                        ],
+                        tensor_fields=["text"] if isinstance(index, UnstructuredMarqoIndex) else None
+
+                    )
+                )
+
+                # Assert that no documents are returned for an empty query
+                res = tensor_search.search(text="", config=self.config, index_name=index.name,
+                                           search_method=SearchMethod.LEXICAL, result_count=10)
+                self.assertIn("hits", res)
+                self.assertEqual(0, len(res['hits']))
+
+    def test_wildcard_lexical_query_structured(self):
```

Comment:
A test description will help a lot. Specifically that you're expecting all results for * but other wildcards are interpreted literally and not supported

Reply:
Added!